UDx-container/Dockerfile_DEB

```diff
@@ -11,83 +11,240 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-###########################################################################
+############################################################################
 
 # Base operating system
 ARG os_version="18.04"
 
-FROM ubuntu:${os_version}
+FROM ubuntu:${os_version} as builder
+
+# useful to keep dpkg from prompting us for information
+ENV DEBIAN_FRONTEND noninteractive
+ENV TERM 1
+
+ENV VERTICA_OPT_DIR="/opt/vertica"
+ENV VERTICA_VOLUME_DIR="/data"
+ENV VERTICA_DATA_DIR="${VERTICA_VOLUME_DIR}/vertica"
+ENV VERTICA_DB_NAME="vsdk"
+
+ARG vertica_db_user="dbadmin"
+ARG vertica_db_group="verticadba"
+ARG vertica_db_name="VMart"
```

Comment:
What is this used for?  We have `VERTICA_DB_NAME` for the database name already, and it refer to a different name of the db "vsdk".